client/state/themes/themes-ui/selectors.js

```diff
@@ -21,3 +21,12 @@ export function getBackPath( state ) {
 	}
 	return `/themes/${ siteSlug }`;
 }
+
+//  Returns true if the theme showcase banner is currently visible
+export function isThemesBannerVisible( state ) {
+	const { themesBannerVisible } = state.themes.themesUI;
+	if ( 'boolean' === typeof themesBannerVisible ) {
+		return themesBannerVisible;
+	}
+	return true;
```

Comment:
Ooh, that looks complicated :sweat_smile: 

Let's just 

```js
export function isThemesBannerVisible( state ) {
	return state.themes.themesUI.themesBannerVisible;
}
```

:smile: 

Rationale: The reducer hierarchy will be initialized at startup, so it's guaranteed to be there. We're setting the `initialState` to `true`, and when closing the banner, to `false`. So this should guarantee us that `state.themes.themesUI.themesBannerVisible` always holds a valid bool.

Reply:
Ha! That's exactly what I had before committing it, so I was on the right path!

What you just mentioned aligns with what I was thinking. I'll dig into it and see what may be going on.